src/graphql/mutations/CancelSessionMutation.graphql

```diff
@@ -0,0 +1,6 @@
+mutation CancelSession($id: ID!) {
+  cancelSession(id: $id) {
```

Comment:
TODO @rschlaefli: This should probably return more than just the status (returns a Session)